apps/redi-connect/src/pages/front/signup/SignUpLanding.scss

```diff
@@ -35,6 +35,9 @@
   }
 }
 
+.signup__type svg {
```

Comment:
Instead of styling svg here, I suggest using [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) properties to properly align items inside the `<div>` with the `className="signup__type"`.